ui/apps/platform/src/Components/ReportJob/MyActiveJobStatusTh.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Flex, FlexItem } from '@patternfly/react-core';
+
+import HelpIconTh from 'Containers/Vulnerabilities/VulnerablityReporting/VulnReports/HelpIconTh';
+
+function MyActiveJobStatusTh() {
+    return (
+        <HelpIconTh
+            popoverContent={
+                <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsMd' }}>
+                    <FlexItem>
+                        <p>
+                            The status of your last requested job from the{' '}
+                            <strong>active job queue</strong>. An <strong>active job queue</strong>{' '}
+                            includes any requested job with the status of <strong>preparing</strong>{' '}
+                            or <strong>waiting</strong> until completed.
+                        </p>
+                    </FlexItem>
+                    <FlexItem>
+                        <p>
+                            <strong>Preparing:</strong>
+                        </p>
+                        <p>Your last requested job is still being processed.</p>
+                    </FlexItem>
+                    <FlexItem>
+                        <p>
+                            <strong>Waiting:</strong>
+                        </p>
+                        <p>
+                            Your last requested job is in the queue and waiting to be processed
+                            since other requested jobs are being processed.
```

Comment:
Not a change request in this contribution, but comment for the future consideration that online dictionary definition for **since** as conjunction has 2 senses (pardon pun) as time and 1 sense as **because**.

Notice that either time or cause are possible in this context.

Reply:
Yeah, the text could be improved. This is just extracting out what we already have in VM Reports. We can work with UX on some improvements. I want to do those later on.